src/app/space/create/codebases/codebases-item-actions/codebases-item-actions.component.less

```diff
@@ -1,2 +1,6 @@
 @import (reference) '../../../../../assets/stylesheets/shared/osio.less';
 .margin-right-20 { margin-right: 20px; }
+
+.remove-codebase-dialog .modal-body {
```

Comment:
break this out as follows:

```
.remove-codebase-dialog {
  .modal-body {
  }
}
```

Reply:
Dialogs were taken from planner ones - https://github.com/fabric8-ui/fabric8-planner/blob/master/src/app/components/work-item-list-entry/work-item-list-entry.component.html#L2
Will update to proper ones, thank you for the links.